src/scripts/breeding/Breeding.ts

```diff
@@ -223,7 +223,11 @@ class Breeding implements Feature {
     }
 
     public gainEggSlot() {
-        this.eggSlots += 1;
+        if (this.eggSlots < this.eggList.length){
+            this.eggSlots += 1;
+        } else {
+            console.error("eggSlots is larger than 4.");
```

Comment:
Magic number, same reasoning as above

Reply:
`eggSlots is larger than eggList.length.`  ok?